DOCS-11373-noninclusiveTerm-v38-duke (#1971)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/hadr-guide.adoc

=== Disaster Recovery with Mule

> Anypoint Cloudhub provides disaster recovery for application and hardware failure by re-deploying the application within the region. If the application uses multiple workers, CloudHub deploys them in separate availability zones within the same region, thereby providing HA across availability zones. The distance between the availability zones is variable and in general it cannot be assumed that they are 350 miles or more apart. If an application uses a single worker, when the availability zone comes down, CloudHub will detect it as part of it's monitoring SYNC process, but it can take up to 30 minutes. CloudHub blocks that availability zone and provisions a new worker in a healthy one. MuleSoft highly recommends to use CloudHub Fabric, with multiple workers for critical applications.

Reviewer: ```suggestion
Anypoint Cloudhub provides disaster recovery for application and hardware failure by re-deploying the application within the region. If the application uses multiple workers, CloudHub deploys them (deploys what application or workers?) in separate availability zones within the same region, thereby providing HA across availability zones. The distance between the availability zones is variable and in general, cannot be assumed to be 350 miles or more apart. If an application uses a single worker when the availability zone comes down, CloudHub will detect it as part of its monitoring SYNC process, but it can take up to 30 minutes. CloudHub blocks that availability zone and provisions a new worker in a healthy one. MuleSoft highly recommends using CloudHub Fabric, with multiple workers for critical applications.
```

Author: review is beyond requested scope
